### DISCUSSION TITLE
Customized install.sh to seperate sqlite-supported and sqlite-unsupported versions

### DISCUSSION THREAD

**Discussion by emrahcom** [ory/kratos#1667] [Show and tell]
I changed [install.sh](https://github.com/ory/kratos/blob/master/install.sh) a bit for my own use case to seperate sqlite-supported and sqlite-unsupported versions. Maybe someone else needs the same...

I added `-s` arg which means "_sqlite supported version_"

```bash
bash <(curl https://raw.githubusercontent.com/emrahcom/emrah-bullseye-templates/main/machines/eb-kratos/root/tools/kratos-download.sh) -b . v0.7.1-alpha.1
./kratos version

bash <(curl https://raw.githubusercontent.com/emrahcom/emrah-bullseye-templates/main/machines/eb-kratos/root/tools/kratos-download.sh) -b . -s v0.7.1-alpha.1
./kratos-sqlite version
```

**Comment by aeneasr** [comment]
Cool stuff, thank you! :) The install scripts definitely need some love...
